src/Razor/src/Microsoft.VisualStudio.RazorExtension/language-configuration.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "comments": {
+    "blockComment": [ "@*", "*@" ]
+  },
+  "brackets": [
+    ["<!--", "-->"],
+    ["<", ">"],
+    ["{", "}"],
+    ["(", ")"]
+  ],
+  "autoClosingPairs": [
```

Comment:
Are we intentionally excluding `<` `>` from auto-closing here because we want:
```html
<button |
```
open-ended behavior?

Reply:
Ya, also if a user types `>` we will auto-complete the end-tag.

Reply:
This might be the same answer to @TanayParikh's question, but what's the difference between an `autoclosingPairs` and an `surroundingPairs`?

Reply:
> This might be the same answer to @TanayParikh's question, but what's the difference between an `autoclosingPairs` and an `surroundingPairs`?

Although I don't believe it's currently implemented by VS "surroundingPairs" allows the IDE to auto-surround pieces of code with specific characters. See: https://code.visualstudio.com/api/language-extensions/language-configuration-guide#autosurrounding

Auto-closing states when you type one of those characters auto-insert the corresponding pair. See: https://code.visualstudio.com/api/language-extensions/language-configuration-guide#autoclosing

Reply:
Any harm in including the other bracket types, specifically `(` `)` under surrounding pairs then?

Reply:
> Any harm in including the other bracket types, specifically `(` `)` under surrounding pairs then?

I don't think so but then again we've never had it in VSCode and things have "just worked" so something else might be at play that we're unaware of